spec/ParseObject.spec.js

```diff
@@ -268,16 +268,28 @@ describe('Parse.Object testing', () => {
     });
   });
 
-  it_exclude_dbs(['postgres'])("can set null", function(done) {
+  it("can set null", function(done) {
```

Comment:
We cannot create a column without a type, so setting null at first is impossible, we'd need to update the test to reflect that.